install_config/configuring_authentication.adoc

```diff
@@ -1306,9 +1626,63 @@ described in the OpenID spec. Must use `https`.
 described in the OpenID spec. Must use `https`.
 ====
 
+. Configure which link:http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[claims] to use: 
++
+----
+claims:
+        id: <4>
+        - custom_id_claim
+        - sub
+        preferredUsername: <5>
+        - preferred_username
+        - email
+        name: <6>
+        - nickname
+        - given_name
+        - name
+        email: <7>
+        - custom_email_claim
+        - email
+----
++
+.. At least one claim to use as the user's identity. The standard identity claim is `sub`.
+
+.. Optionally, indicate which claims to use as the user's preferred user name,
+display name, and email address. If multiple claims are specified, the first one
+with a non-empty value is used. The standard claims are:
++
+[horizontal]
+`preferred_username`:: The preferred user name when provisioning a user.
+`email`:: Email address.
+`name`:: Display name.
+
+. Restart the {product-title} services for the changes to take effect:
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master
+----
+endif::[]
+
+[[open_id_connect_full]]
+*Configuring Full Master Configuration*
```

Comment:
this is rendering in a monospace font, which looks really weird to me for a section header